src/Microsoft.Graph.Core/Requests/Content/BatchRequestContent.cs

```diff
@@ -210,7 +210,7 @@ private string GetRelativeUrl(Uri requestUri)
 
         protected override async Task SerializeToStreamAsync(Stream stream, TransportContext context)
         {
-            using (StreamWriter streamWritter = new StreamWriter(stream))
+            using (StreamWriter streamWritter = new StreamWriter(stream, new UTF8Encoding(), 1024, true))
```

Comment:
We need to dispose the stream now that the StreamWriter is keeping it open.

Reply:
HttpContent will [dispose](https://github.com/dotnet/corefx/blob/d69d441dfb0710c2a34155c7c4745db357b14c96/src/System.Net.Http/src/System/Net/Http/HttpContent.cs#L539 ) the stream for us once the request has been sent. The target stream is used internally by HttpClient ([HttpConnection.cs](https://github.com/dotnet/corefx/blob/3c13f6e53bd57db3a75c8ddbbf2be6b3b6b7c934/src/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/HttpConnection.cs)) and we only need to write our content to it and keep it open.